src/main/java/space/obminyashka/items_exchange/controller/UserController.java

```diff
@@ -42,13 +42,14 @@
 @Slf4j
 public class UserController {
 
+    private final UserService userService;
```

Comment:
You need to revert such changes. You just moved the classes and erased previous commit history for such classes.